src/components/checkout.js

```diff
@@ -0,0 +1,48 @@
+import React from "react"
+const buttonStyles = {
+  fontSize: "16px",
+  textAlign: "center",
+  color: "#fff",
+  outline: "none",
+  padding: "12px 60px",
+  boxShadow: "2px 5px 10px rgba(0,0,0,.1)",
+  backgroundColor: "rgb(255,0,0)",
+  borderRadius: "6px",
+  letterSpacing: "1.5px",
+}
+const Checkout = class extends React.Component {
+  // Initialise Stripe.js with your publishable key.
+  // You can find your key in the Dashboard:
+  // https://dashboard.stripe.com/account/apikeys
+  componentDidMount() {
+    this.stripe = window.Stripe("pk_test_XiirziufElakjoOpyuyCrPfo")
+  }
+  async redirectToCheckout(event) {
+    event.preventDefault()
+    const { error } = await this.stripe.redirectToCheckout({
+      items: [
+        { sku: "sku_Fvit7rtTpQFLdF", quantity: 1, name: "GOLD" },
```

Comment:
You should only have one item here, and its sku should be `this.props.sku` and its name `this.props.name`.

As the code stands now the user will buy all three.

Reply:
Ok 😺